CHANGES.md

```diff
@@ -30,6 +30,8 @@
 
 - `--diff` correctly indicates when a file doesn't end in a newline (#1662)
 
+- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules (#1780)
```

Comment:
```suggestion
- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules
  (#1780)
```
Fix lint. Sorry for the churn, but while we're at it we might as well improve the wording.